Lint检查失败: primary-node.ts 中 async 方法缺少 await 表达式

## 测试输出错误信息

```
/Users/hs3180/Documents/disclaude/src/nodes/primary-node.ts
  173:7   error    Async method 'sendToRemoteNode' has no 'await' expression            require-await
  201:9   error    Async method 'routeCardAction' has no 'await' expression             require-await
  751:3   error    Async method 'routeCardAction' has no 'await' expression             require-await
  763:3   error    Async method 'sendCardActionToRemoteNode' has no 'await' expression  require-await

✖ 93 problems (4 errors, 89 warnings)
```

## 根本原因分析

在 PR #956 (`fix(nodes): enable Worker Node card action routing via WebSocket`) 中新增的异步方法声明为 `async`，但方法体内没有使用 `await` 关键字。

涉及的方法:
- `sendToRemoteNode` (line 173)
- `routeCardAction` (line 201, 751)
- `sendCardActionToRemoteNode` (line 763)

这些方法可能是为了将来扩展而声明为 async，或者返回 Promise 但不需要 await。ESLint 的 `require-await` 规则要求 async 方法必须包含 await 表达式。

## 建议修复方案

1. 如果方法不需要 await，移除 `async` 关键字并直接返回值
2. 如果方法需要返回 Promise，使用 `Promise.resolve()` 包装返回值
3. 如果确实需要异步操作，添加适当的 await 调用

## 触发提交

- 47ebddf fix(nodes): enable Worker Node card action routing via WebSocket (#956)
- ea95ca2 fix(pilot): restore session context from persisted chat history on restart (#957)

--- a/src/nodes/primary-node.ts
+++ b/src/nodes/primary-node.ts
@@ -170,7 +170,7 @@ export class PrimaryNode extends EventEmitter {
 
     // Issue #935: Initialize CardActionRouter for Worker Node card routing
     this.cardActionRouter = new CardActionRouter({
-      sendToRemoteNode: async (nodeId: string, message: CardActionMessage) => {
+      sendToRemoteNode: (nodeId: string, message: CardActionMessage) => {
         return this.sendCardActionToRemoteNode(nodeId, message);
       },
       isNodeConnected: (nodeId: string) => {
@@ -198,7 +198,7 @@ export class PrimaryNode extends EventEmitter {
         appId,
         appSecret,
         // Issue #935: Route card actions to Worker Nodes
-        routeCardAction: async (message) => {
+        routeCardAction: (message) => {
           return this.routeCardAction({
             type: 'card_action',
             chatId: message.chatId,
@@ -748,7 +748,7 @@ export class PrimaryNode extends EventEmitter {
    * @param message - Card action message to route
    * @returns True if the action was routed to a Worker Node, false otherwise
    */
-  async routeCardAction(message: CardActionMessage): Promise<boolean> {
+  routeCardAction(message: CardActionMessage): Promise<boolean> {
     return this.cardActionRouter.routeCardAction(message);
   }
 
@@ -760,25 +760,25 @@ export class PrimaryNode extends EventEmitter {
    * @param message - Card action message to send
    * @returns True if the message was sent successfully
    */
-  private async sendCardActionToRemoteNode(nodeId: string, message: CardActionMessage): Promise<boolean> {
+  private sendCardActionToRemoteNode(nodeId: string, message: CardActionMessage): Promise<boolean> {
     const node = this.execNodeRegistry.getNode(nodeId);
     if (!node || node.isLocal || !node.ws) {
       logger.warn({ nodeId }, 'Remote node not found or not connected');
-      return false;
+      return Promise.resolve(false);
     }
 
     if (node.ws.readyState !== WebSocket.OPEN) {
       logger.warn({ nodeId }, 'Remote node WebSocket not open');
-      return false;
+      return Promise.resolve(false);
     }
 
     try {
       node.ws.send(JSON.stringify(message));
       logger.debug({ nodeId, chatId: message.chatId }, 'Card action sent to remote Worker Node');
-      return true;
+      return Promise.resolve(true);
     } catch (error) {
       logger.error({ err: error, nodeId }, 'Failed to send card action to remote Worker Node');
-      return false;
+      return Promise.resolve(false);
     }
   }
 

